svd/atmega4809.svd

```diff
@@ -0,0 +1,18300 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The SVD files are not meant to be checked in.  That's why `svd/` is excluded in `.gitignore` :)